cypress/e2e/hermioneAccount.spec.js

```diff
@@ -1,11 +1,98 @@
 /// <reference types='cypress' />
+import { faker } from '@faker-js/faker';
 
 describe('Bank app', () => {
-  before(() => {
-    cy.visit('/');
+  beforeEach(() => {
+    cy.visit('');
   });
 
+  const userHermiona = 'Hermoine Granger';
+  const accounNumber = 1001;
```

Comment:
The variable name `accounNumber` seems to have a typo. It should likely be `accountNumber`.